app/javascript/dashboard/api/helpCenter/articles.js

```diff
@@ -14,14 +14,20 @@ class ArticlesAPI extends PortalsAPI {
     status,
     author_id,
     category_slug,
+    sort,
   }) {
     let baseUrl = `${this.url}/${portalSlug}/articles?page=${pageNumber}&locale=${locale}`;
     if (status !== undefined) baseUrl += `&status=${status}`;
     if (author_id) baseUrl += `&author_id=${author_id}`;
     if (category_slug) baseUrl += `&category_slug=${category_slug}`;
+    if (sort) baseUrl += `&sort=${sort}`;
     return axios.get(baseUrl);
```

Comment:
Let's refactor this to use `URLSearchParams` standard

```js
const queryParams = new URLSearchParams({ page: pageNumber, locale});

if (status !== undefined) queryParams.set('status', status);
if (author_id) queryParams.set('author_id', author_id);
if (category_slug) queryParams.set('category_slug', category_slug);
if (sort) queryParams.set('sort', sort);

const baseUrl = `${this.url}/${portalSlug}/articles?${queryParams.toString()}`;
return axios.get(baseUrl);
```

Would be much cleaner and easy to work with